src/main/java/net/tiagofar78/prisonescape/PrisonEscape.java

```diff
@@ -30,6 +31,12 @@ public void onEnable() {
 			saveDefaultConfig();
 		}
 		
+        World world = Bukkit.getWorld("PrisonEscape");
```

Comment:
You should not do this kind of thing in this file. This is a startup file!

This also doesn't solve the problem, it will just turn off rain for that moment. You have to go Events file and add the event of weather change (or something like that) and cancel it.

Reply:
ah okay mb